www/templates/diary/trip_list_item.html

```diff
@@ -2,16 +2,26 @@
 <!-- New Trip Template format  -->
 
     <ion-item class="unified-diary-item" style="background-color: transparent;" class="list-item">
-        <div class="diary-card">
+        <div class="diary-card" ng-class="[diary-card, {draft: trip.key.includes('UNPROCESSED')}]">
             <div class="start-time-tag">
                 <b>{{trip.display_start_time}}</b>
                 <i ng-if="trip.display_start_date_abbr">&nbsp;({{trip.display_start_date_abbr}})</i>
             </div>
             <!-- Leaflet Map -->
             <div id="left-diary-card" class="diary-map-shell" ng-click="showDetail($event, trip)">
-                <leaflet class="diary-map" geojson="trip" defaults="mapCtrl.defaults"
-                    id="$index" data-tap-disabled="true" height="{{mapHeight}}">
-                </leaflet>
+                <!-- Modes / Percentages -->
+                <div class="diary-modes-percents" ng-if="trip.percentages">
+                    <span
+                        class="diary-modes-percents-text"
+                        ng-repeat="sectionPct in trip.percentages" style="color: {{ sectionPct.color }}">
+                        <span class="diary-modes-percents-icon {{sectionPct.icon}}"></span>
```

Comment:
I did want to discuss this at some point.

Icons seem a lot nicer to keep the interface looking clean and comprehensible to users - so if possible, I think it'd be nice to use icons on the cards, and then follow up with icons + text in the details page.

How many modes does OpenStreetMap have exactly? Isn't there a fixed number? It _seems_ like we should be able to map every mode to an icon. I know you mentioned that ferry was an issue before - don't we have a boat icon?
Even if two modes end up using the same/similar icon, that wouldn't be catastrophic as long as the details page disambiguates with text. Soon enough, we should be moving to a richer icons library that would allow us to express more modes of transport.

Reply:
Once you restore to the original list, https://github.com/e-mission/e-mission-phone/pull/974#discussion_r1207651398 we can do done wrt this change, and then work on a redesign as part of a bunch of other cleanups

Reply:
addressed in #980 - restored the MotionTypes from before, plus a few others

We will still talk design for icons vs. text at a later time.